crate/cli/src/actions/shared/unwrap_key.rs

```diff
@@ -49,6 +49,16 @@ pub struct UnwrapKeyAction {
     )]
     unwrap_password: Option<String>,
 
+    /// An optional hexadecimal salt to derivate the password into a key.
+    #[clap(
+        long = "passwd-salt",
```

Comment:
```suggestion
        long = "password-salt",
```

Reply:
commit 8c674423862328a80812281746a4ff4a9cb50926 should fix this.